src/shared_utilities/utils.jl

```diff
@@ -273,3 +274,8 @@ function NonInterpSavingCallback(saved_values, saveat::Vector{FT}) where {FT}
         save_positions = (false, false),
     )
 end
+
+function FTfromY(Y::ClimaCore.Fields.FieldVector)
```

Comment:
can we call this `FT_from_Y`? I think that better matches the convention we use for function naming

Reply:
I went with `FTfromY` because the `Julia` conventions recommend not using underscores when the name is short and clearly readable (as in this case, which is helped by the capital letters). 

I can change to `FT_from_Y` if you prefer.

Reply:
Okay, that's fine!